qiskit_experiments/library/randomized_benchmarking/rb_utils.py

```diff
@@ -1,319 +0,0 @@
-            # might happen if the backend has no properties (e.g. qasm simulator)
-            return None
-        return error_dict
-
-    @staticmethod
-    @deprecated_function(
-        last_version="0.4",
-        msg=(
-            "Now this method is integrated into 'StandardRB._transpiled_circuits' method. "
-            "You don't need to explicitly call this method."
-        ),
-    )
-    def count_ops(
-        circuit: QuantumCircuit, qubits: Optional[Sequence[int]] = None
-    ) -> Dict[Tuple[Sequence[int], str], int]:
-        """Counts occurrences of each gate in the given circuit
-
-        Args:
-            circuit: The quantum circuit whose gates are counted
-            qubits: A list of qubits to filter irrelevant gates
-
-        Returns:
-            A dictionary of the form (qubits, gate) -> value where value
-            is the number of occurrences of the gate on the given qubits
-        """
-        if qubits is None:
-            qubits = range(len(circuit.qubits))
-        count_ops_result = {}
-        for instr, qargs, _ in circuit:
-            instr_qubits = []
-            skip_instr = False
-            for qubit in qargs:
-                qubit_index = circuit.qubits.index(qubit)
-                if qubit_index not in qubits:
-                    skip_instr = True
-                instr_qubits.append(qubit_index)
-            if not skip_instr:
-                instr_qubits = tuple(instr_qubits)
-                count_ops_result[(instr_qubits, instr.name)] = (
-                    count_ops_result.get((instr_qubits, instr.name), 0) + 1
-                )
-        return count_ops_result
-
-    @staticmethod
-    @deprecated_function(last_version="0.4")
-    def gates_per_clifford(
-        ops_count: List,
-    ) -> Dict[Tuple[Sequence[int], str], float]:
-        """
-        Computes the average number of gates per clifford for each gate type
-        in the input from raw count data coming from multiple circuits.
-
-        Args:
-            ops_count: A List of [key, value] pairs where
-              key is [qubits, gate_name] and value is the average
-              number of gates per clifford of the type for the given key
-
-        Returns:
-            A dictionary with the mean value of values corresponding
-            to key for each key.
-
-        """
-        result = {}
-        for ((qubits, gate_name), value) in ops_count:
-            qubits = tuple(qubits)  # so we can hash
-            # for each (qubit, gate name) pair collect
-            # (number of gates, number of cliffords)
-            if (qubits, gate_name) not in result:
-                result[(qubits, gate_name)] = [0, 0]
-            result[(qubits, gate_name)][0] += value[0]
-            result[(qubits, gate_name)][1] += value[1]
-        return {key: value[0] / value[1] for (key, value) in result.items()}
-
-    @staticmethod
-    def coherence_limit(nQ=2, T1_list=None, T2_list=None, gatelen=0.1):
```

Comment:
This is not deprecated yet. I'm not sure if this is the right place to put this stuff though. This calculation is often used with gate-level benchmark experiment such as RB or QPT to roughly analyze error budget, but we don't have util directory.